hscript/Interp.hx

```diff
@@ -562,7 +562,8 @@ class Interp {
 			return (v : Array<Dynamic>).iterator();
 		if( v.iterator != null ) v = v.iterator();
 		#else
-		try v = v.iterator() catch( e : Dynamic ) {};
+		#if (cpp) if ( v.iterator != null ) #end
+			try v = v.iterator() catch( e : Dynamic ) {};

```

Comment:
Is the try/catch necessary on cpp after the nullcheck? If not, this could go with the flash/php #if above

Reply:
not sure, I left it on to be safe.